Sources/BraintreeCoreSwift/BTAPIClientHTTPType.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// :nodoc:
+@objc public enum BTAPIClientHTTPTypeSwift: Int {
```

Comment:
This needs to be public since it is used in the get and post public functions. Those functions are used by other modules in the SDK to make requests.